src/index.html

```diff
@@ -2,15 +2,44 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="container bigger-search-bar"
+    action=""
+    method="post"
+    data-qa="big"
+    >
+      <img src="./images/Search.svg"
+      alt="search-loop"
+      class="big-icon">
+      <input
+      class="bigger-input"
+      type="text"
+      name="big-search-bar"
+      placeholder="Try &quot;Los Angeles&quot;"
+      data-qa="keypress"
+      >
```

Comment:
```suggestion
      <input
        class="bigger-input"
        type="text"
        name="big-search-bar"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```
Не забывай про отступы